pennylane/qchem/structure.py

```diff
@@ -227,10 +229,12 @@ def excitations(electrons, orbitals, delta_sz=0):
             ``sz[p] + sz[p] - sz[r] - sz[s] = delta_sz`` for the spin-projection ``sz`` of
             the orbitals involved in the single and double excitations, respectively.
             ``delta_sz`` can take the values :math:`0`, :math:`\pm 1` and :math:`\pm 2`.
+        fermionic (bool): Return a list of ``FermiWord`` objects instead of a list of lists
```

Comment:
Please update to this or a similar statement:

fermionic (bool): Return a list of ``FermiWord`` objects instead of the list of orbital indices, if set to ``True``. Default is ``False``.